api/src/main/java/org/pmiops/workbench/api/WorkspaceAdminController.java

```diff
@@ -59,13 +59,9 @@ public WorkspaceAdminController(WorkspaceAdminService workspaceAdminService) {
   @Override
   @AuthorityRequired({Authority.RESEARCHER_DATA_VIEW})
   public ResponseEntity<ReadOnlyNotebookResponse> adminReadOnlyNotebook(
-      String workspaceNamespace,
-      String workspaceName,
-      String notebookName,
-      AccessReason accessReason) {
+      String workspaceNamespace, String notebookName, AccessReason accessReason) {
```

Comment:
Workspace name (firecloud name, no spaces) is not strictly necessary, and is also not immediately available to the UI page

Reply:
interesting.